test/actor_test.py

```diff
@@ -1906,6 +1906,36 @@ def method(self):
         # we should also test this from a different driver.
         ray.get(new_f.method.remote())
 
+    def testRegisterAndGetActorHandle(self):
```

Comment:
can we change this to `testRegisterAndGetNamedActors`? It'll be easier to find by searching